modules/oe_content_person/modules/oe_content_person_sub_entity_reference/oe_content_person_sub_entity_reference.info.yml

```diff
@@ -0,0 +1,19 @@
+name: OpenEuropa Content Person Sub Entity Reference
+description: Contains Person reference field storage and the Person sub-entity bundle Person.
+package: OpenEuropa
+type: module
+core_version_requirement: ^9.4 || ^10
+
+dependencies:
+  - oe_content:oe_content_person
+  - oe_content:oe_content_sub_entity_person
+
+config_devel:
+  install:
+    - core.entity_form_display.oe_person.oe_person.default
+    - core.entity_view_display.oe_person.oe_person.default
+    - field.field.oe_person.oe_person.oe_node_reference
+    - field.storage.oe_person.oe_node_reference
+    - oe_content_sub_entity_person.oe_person_type.oe_person
+  optional:
+    - language.content_settings.oe_person.oe_person
```

Comment:
These configs were moved from the module, they need to be revisited and the dependencies inside these configs need to be updated to the new ones.

Reply:
Hi @nagyad,

I removed the config_devel list of configs that were moved to tasks/install in [4813864](https://github.com/openeuropa/oe_content/pull/591/commits/48138644c88a2c756c9ef486586e7efb38f41d58).  I double checked the configs and dependencies and it looked good to me (I also did a rebase and a re-export of these configs but didn't see any difference).

Reply:
I meant the config files with those names.